apps/meteor/tests/end-to-end/api/livechat/18-rooms-ee.ts

```diff
@@ -254,4 +254,39 @@ import { IS_EE } from '../../../e2e/config/constants';
 			expect(updatedRoom).to.not.have.property('onHold');
 		});
 	});
+
+	describe('visitor abandonment feature', () => {
+		before(async () => {
+			await updateSetting('Livechat_abandoned_rooms_action', 'Livechat_close_chat');
+			await updateSetting('Livechat_visitor_inactivity_timeout', 60);
+		});
+
+		it('should set predictedVisitorAbandonmentAt when agent sends a message', async () => {
```

Comment:
add a test to check that the ts is not updated on agent's subsequent messages 👀

Reply:
Done :heavy_check_mark: